src/Microsoft.Restier.EntityFramework/ApiBuilderExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+#if EF7
+using Microsoft.Data.Entity;
+#else
+using System.Data.Entity;
+#endif
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Restier.Core;
+using Microsoft.Restier.Core.Model;
+using Microsoft.Restier.Core.Query;
+using Microsoft.Restier.Core.Submit;
+using Microsoft.Restier.EntityFramework.Model;
+using Microsoft.Restier.EntityFramework.Query;
+using Microsoft.Restier.EntityFramework.Submit;
+
+namespace Microsoft.Restier.EntityFramework
+{
+    [CLSCompliant(false)]
+    public static class ApiBuilderExtensions
+    {
+        public static IServiceCollection UseDbContext<T>(this IServiceCollection obj)
```

Comment:
Add multiple DbContext type is also possible, just need modifications to DB related services.
Of course we should first ask "do we need"?